### PR TITLE
Ticket/1

### PR DESCRIPTION
**Pull request by tanialopezab**
I've focused face on the page. Please check the task

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,4 +13,4 @@
             </div>
         </div>        
     </body>
-</html>
```
Comment (rodriguezmontesinoseva) [inline index.html]: There are two htm tags

File: index.html [hunk 2]
```diff
@@ -13,4 +13,4 @@
             </div>
```
Comment (rodriguezmontesinoseva) [inline index.html]: the face isn't vertically centered

File: index.html (hunk 1 above)
Comment (tanialopezab) [reply index.html]: ok it' s done

File: index.html (hunk 2 above)
Comment (tanialopezab) [reply index.html]: ok it 's done

**Comment by tanialopezab** [comment]
delete branch ticket/1 after merge with dev branch

**Pull request merged**